cds_ils/cli.py

```diff
@@ -223,6 +231,38 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index opendefinition opendefinition --force")
 
 
+def recreate_cover(pid, record_class):
+    """Recreate cover for a given record pid."""
+    record = record_class.get_record_by_pid(pid)
+    if record:
+        pick_identifier_with_cover_task(record)
+        click.secho("Recreated cover for PID: {}".format(pid), fg='blue')
+    else:
+        click.secho("No document or series found with PID: {}", fg='red')
+
+
+@covers.command()
+@click.option('-t', '--pid-type', required=True,
+              type=click.Choice(['docid', 'serid']))
+@click.option('-a', '--all', required=False, is_flag=True)
+@click.option('-p', '--pid')
+@with_appcontext
+def recreate(pid, pid_type, all):
+    """Recreate covers command."""
+    if pid_type == 'docid':
+        record_class, search_class, indexer_class = \
+            Document, DocumentSearch, DocumentIndexer
```

Comment:
I would use the ones configured in the config instead of hardcoding, whenever possible:
```suggestion
            current_app_ils.document_record_cls,....
```